packages/ra-ui-materialui/src/input/AutocompleteInput.spec.tsx

```diff
@@ -935,4 +938,35 @@ describe('<AutocompleteInput />', () => {
             screen.getByText('No options');
         });
     });
+
+    it('should not display "Dalmatian #1001" when given no `perPage` parameters and a list of 1001 dalmatians', async () => {
+        render(<VeryLargeOptionsNumber includeAdmin={false} />);
+
+        const input = screen.getByLabelText('Dalmatians', {
+            selector: 'input',
+        }) as HTMLInputElement;
+        fireEvent.focus(input);
+        expect(screen.queryByText('Dalmatian #1001')).toBeNull();
+    });
+
+    it('should display "Dalmatian #1001" when given `perPage=1001` parameters and a list of 1001 dalmatians', async () => {
```

Comment:
In the test description, speak about the feature you're testing, not the dataset you're using